pkg/policy/api/icmp.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import "strconv"
+
+type ICMPRules []ICMPRule
+
+// ICMPRule is a list of ICMP-type.
+type ICMPRule struct {
+	// Types is a list of ICMP-type.
+	//
+	// +kubebuilder:validation:Optional
+	Types []ICMPType `json:"types,omitempty"`
+}
+
+// ICMPType is a ICMP-type.
+type ICMPType struct {
```

Comment:
Just curious why we've chosen to call this `ICMPType` rather than `ICMPRule`.

Reply:
What do you mean? I use `ICMPRule` in the line above.

Reply:
Ah, nevermind. :)